pkg/apis/intevents/v1alpha1/topic_validation.go

```diff
@@ -55,16 +57,16 @@ func (current *Topic) CheckImmutableFields(ctx context.Context, original *Topic)
 		return nil
 	}
 
-	var errs *apis.FieldError
-	// Topic is immutable.
-	if original.Spec.Topic != current.Spec.Topic {
-		errs = errs.Also(
-			&apis.FieldError{
-				Message: "Immutable field changed",
-				Paths:   []string{"spec", "topic"},
-				Details: fmt.Sprintf("was %q, now %q", original.Spec.Topic, current.Spec.Topic),
-			})
+	if diff := cmp.Diff(original.Spec, current.Spec,
+		cmpopts.IgnoreFields(TopicSpec{}, "PropagationPolicy", "EnablePublisher")); diff != "" {
```

Comment:
Same as above... I added the topic immutable check logic to all versions to make them consistent.

Reply:
thanks!